src/style.css

```
@@ -1 +1,242 @@
-/* empty source */
+body {
+  --card-max-width: 198px;
+  --card-border: var(1px);
+  --card-border-radius: 5px;
+  --min-section-gap: 16px;
+  --min-section-top-gap: 16px;
+  --card-header-top-gap: 32px;
+  --card-margin: 0;
+  --card-padding: 0;
+  --card-padding-top: 40px;
+  --card-image-height: 134px;
+  --card-font-size: 12px;
+  --cardfont-color: #060b35;
+}
+
+article {
```

Try not to use tag selectors except body and html. Use class selectors instead

---

Not fixed. Please fix all the tag selectors
```suggestion
.card {
```